packages/sdk-react/src/contexts/TurnkeyContext.tsx

```diff
@@ -27,14 +27,12 @@ export const TurnkeyProvider: React.FC<TurnkeyProviderProps> = ({
   config,
   children,
 }) => {
-  const [turnkeyClient, setTurnkeyClient] = useState<
-    TurnkeyBrowserSDK | undefined
+  const [turnkey, setTurnkey] = useState<Turnkey | undefined>(undefined);
```

Comment:
minor nit: i'm down to initialize these as empty `const [turnkey, setTurnkey] = useState<Turnkey | undefined>();`

Reply:
what's the difference? I honestly don't know. feels like it would be better to be explicit about the initial value though?

Reply:
no difference! down for either. let's ship it 🚢